core-java-modules/core-java-14/src/main/java/com/baeldung/java14/textblocks/TextBlocks14.java

```diff
@@ -0,0 +1,16 @@
+package com.baeldung.java14.textblocks;
+
+public class TextBlocks14 {
```

Comment:
Indentation looks off here in this class?

Reply:
Residu of copy-pasting :-)

Reply:
I see Eclipse refuses to format because I did not setup the JDK :-(